src/game/client.cpp

```diff
@@ -477,11 +477,14 @@ namespace client
                 items.emplace_back(v >= W_OFFSET && v < W_ITEM ? v : 0);
             }
         }
-        game::player1.loadweap.shrink(0);
-        loopv(items) if(game::player1.loadweap.find(items[i]) < 0)
-        {
-            game::player1.loadweap.add(items[i]);
-            if(game::player1.loadweap.length() >= W_LOADOUT) break;
+        game::player1.loadweap.clear();
+        for (const auto &i : items) {
```

Comment:
The use of reference adds overhead for integral numeric types, as `i` needs to be dereferenced before use, and the pointer consumes additional memory (an overhead of 8 bytes on a 64-bit system compared to the typical 4 bytes for an `int`). It is more efficient to copy here.

Reply:
True that, solving